frontend/public/locales/en/demographics-oral-health-questions.json

```diff
@@ -15,24 +15,43 @@
     "question4": "4. Do you identify as a person with a disability?",
     "question4-note": "A person with a disability is a person who has a long-term difficulty or  condition, such as vision, hearing, mobility, flexibility, dexterity, pain, learning, developmental, memory or mental health-related impairments, that limit their daily activities inside or outside the home such as at school, work, or in the community in general.",
 
-    "button-back": "Back < ",
+    "button-back": " < Back",
     "button-continue": "Continue > "
   },
   "part2": {
     "page-title": "Voluntary demographic and oral health questions part 2 of 2",
-
+    "breadcrumbs": {
+      "canada-ca": "Canada.ca",
+      "benefits": "Benefits",
+      "dental-coverage": "Dental coverage",
+      "canadian-dental-care-plan": "Canadian Dental Care Plan"
+    },
     "question-gender": "5. What is your gender? Gender refers to current gender which may be different from sex assigned at birth and may be different from what is indicated on legal documents.",
     "question-gender-other-specify": "Specify",
     "question-mouth-pain": "6. In the past 12 months, how often have you had any persistent or on-going mouth pain? Mouth pain includes pain in teeth, gums, tongue, jaw, jaw joints.",
     "question-last-dental-visit": "7. When was the last time you saw a dentist, denturist, dental hygienist or any other dental specialist? Services may have been provided in any setting where the oral health professional is licensed to practice",
     "question-avoided-dental-cost": "8. In the past 12 months, have you avoided going to a dental professional for dental care due to the cost of care?",
 
-    "button-back": "Back < ",
+    "button-back": " < Back",
```

Comment:
We should be using icons instead (maybe in a future PR since this isn't the only page with this issue).

Example: https://github.com/DTS-STN/canadian-dental-care-plan/blob/8655134344c1156a93568c3f3fd3c0571e5953a9/frontend/app/routes/_public%2B/apply%2B/%24id%2B/terms-and-conditions.tsx#L124

Reply:
I've corrected those icons for the back and continue button